inventories/group_vars/control-plane/metal.yml

```diff
@@ -12,16 +12,16 @@ metal_api_admin_key: metal-admin
 metal_api_nsq_tcp_address: nsqd:4150
 
 metal_api_images:
-- id: firewall-ubuntu-2.0.20201126
-  name: Firewall 2 Ubuntu 20201126
-  description: Firewall 2 Ubuntu 20201126
-  url: https://images.metal-stack.io/metal-os/master/firewall/2.0-ubuntu/20201126/img.tar.lz4
+- id: firewall-ubuntu-2.0.20201214
```

Comment:
This image is broken

Reply:
Mh... it's not so good that we are releasing broken images. :( 

Proper releases with notes and things would be nice to for the images.

Will revert this.

Reply:
Sadly we have no proper release process for images.